### ISSUE TITLE
BL1430 support

### ISSUE THREAD

**Issue by pety3bi** [mnh-jansson/open-battery-information#6]
As you can see, BL1430 is also can be read, is it possible to make changes for this model, so it can identify only 4 actual cells for this types of batteries?

![IMG_20240804_202356_507](https://github.com/user-attachments/assets/4af0b9bc-0b8c-47a5-aa5d-0bf994ed1e21)

**Comment by mnh-jansson** [comment]
Thanks for reporting! I'll add another check for the model to adjust to 4 cell. 

If possible, pictures of the BMS would be helpful for the wiki.

**Comment by nvn887** [comment]
wrong cell voltage 5 , BL1440
![vidu 1 sai dien ap](https://github.com/user-attachments/assets/f21ded24-8c58-4f5f-a9c6-c2dfe45ae236)
![DSC_0007](https://github.com/user-attachments/assets/761e8349-df55-4792-b37a-cae32b16d2b2)

**Comment by mnh-jansson** [comment]
> wrong cell voltage 5 , BL1440
> 
> ![vidu 1 sai dien ap](https://github.com/user-attachments/assets/f21ded24-8c58-4f5f-a9c6-c2dfe45ae236)
> 
> ![DSC_0007](https://github.com/user-attachments/assets/761e8349-df55-4792-b37a-cae32b16d2b2)
> 
> 

Thanks! Can you confirm that unlock works?

**Comment by nvn887** [comment]
it's not broken, just reading the wrong voltage.

**Comment by mnh-jansson** [comment]
@nvn887 I get that, but i was wondering if you also tried to Clear errors and if that worked.

**Comment by nvn887** [comment]
yes, i tried reset (even though it was not an error) but it didn`t work

https://github.com/user-attachments/assets/1a19be25-ee82-4ab7-a82a-ac171711b522
